plugins/dell-kestrel/fu-dell-kestrel-ec.c

```diff
@@ -73,6 +73,23 @@ fu_dell_kestrel_ec_is_dev_present(FuDellKestrelEc *self,
 	return dev_entry != NULL;
 }
 
+gboolean
+fu_dell_kestrel_ec_is_chunk_supported(FuDevice *device, FuDellKestrelEcDevType dev_type)
+{
+	switch (dev_type) {
+	case FU_DELL_KESTREL_EC_DEV_TYPE_RMM:
+		return FALSE;
+	case FU_DELL_KESTREL_EC_DEV_TYPE_PD:
+		FuDellKestrelEc *self = FU_DELL_KESTREL_EC(device);
```

Comment:
you need to declare this at the start of the function I believe.

Reply:
Moved to the top, thanks.